activestorage/app/models/active_storage/blob.rb

```diff
@@ -201,8 +201,8 @@ def representable?
   # with users. Instead, the +service_url+ should only be exposed as a redirect from a stable, possibly authenticated URL.
   # Hiding the +service_url+ behind a redirect also gives you the power to change services without updating all URLs. And
   # it allows permanent URLs that redirect to the +service_url+ to be cached in the view.
-  def service_url(expires_in: service.url_expires_in, disposition: "inline")
-    service.url key, expires_in: expires_in, disposition: disposition, filename: filename, content_type: content_type
+  def service_url(expires_in: service.url_expires_in, disposition: :inline)
+    service.url key, expires_in: expires_in, disposition: forced_disposition || disposition, filename: filename, content_type: content_type
```

Comment:
What do you think about hoisting the conditional into this method, since we’re not returning more than one override from `forced_disposition`?

```ruby
service.url key, expires_in: expires_in, disposition: forcibly_serve_as_binary? ? :attachment : disposition, filename: filename, content_type: content_type
```

Reply:
Sounds good to me, it looks long but I think it's clearer 👍